.github/workflows/periodic_pagebench.yml

```
@@ -43,6 +43,14 @@ jobs:
       AWS_DEFAULT_REGION : "eu-central-1"
       AWS_INSTANCE_ID : "i-02a59a3bf86bc7e74"
     steps:
+    # we don't need the neon source code because we run everything remotely
+    # however we still need the local github actions to run the allure step below
+    - name: Checkout
+      uses: actions/checkout@v4
```

I think the name is clear for this action here, so we can use just

```suggestion
    - uses: actions/checkout@v4
```